src/lib/dnssd/minimal_mdns/core/tests/TestRecordWriter.cpp

```diff
@@ -17,16 +17,19 @@
  */
 
 #include <lib/dnssd/minimal_mdns/core/RecordWriter.h>
-#include <lib/support/UnitTestRegistration.h>
+//#include <lib/support/UnitTestRegistration.h>
```

Comment:
delete commented out text instead of just commenting it out. If we need history, we can use git history. Applies throughout.